clang/test/Analysis/ptr-arith.cpp

```diff
@@ -165,3 +165,124 @@ void LValueToRValueBitCast_dumps(void *p, char (*array)[8]) {
 unsigned long ptr_arithmetic(void *p) {
   return __builtin_bit_cast(unsigned long, p) + 1; // no-crash
 }
+
+
+void escape(int*);
+
+struct AllocOpaqueFlag {};
+
+void* operator new(unsigned long, void *ptr) noexcept { return ptr; }
+void* operator new(unsigned long, void *ptr, AllocOpaqueFlag const&) noexcept { return ptr; }
```

Comment:
I was expecting the AllocOpaqueFlag overloads to be opaque.

Reply:
C&P mistake. Made them opaque, which makes one of the warning disappear (as it probably should).